.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: Report a bug encountered with Scylla Operator
+labels: kind/bug
+body:
+- type: textarea
+  id: problem
+  attributes:
+    label: What happened?
+    description: |
+      Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+  validations:
+    required: true
+
+- type: textarea
+  id: expected
+  attributes:
+    label: What did you expect to happen?
+  validations:
+    required: true
+
+- type: textarea
+  id: repro
+  attributes:
+    label: How can we reproduce it (as minimally and precisely as possible)?
```

Comment:
```suggestion
   label: How can we reproduce it? Provide the fewest and most precise steps possible.
```
nit

Reply:
it's identical to the kube issue tracker

Reply:
I think overusing the parentheses obfuscates the text, but I don't feel strong about it. On a side note, I don't see it being identical to the wording in kube repo as a convincing argument - in terms of code, sure, but stuff like this doesn't have to be some golden standard there.